src/__experimental__/components/radio-button/radio-button.stories.mdx

```diff
@@ -123,6 +123,39 @@ A `ml` prop can be supplied to align the `RadioButtonGroup` with above and below
   </Story> 
 </Preview>
 
+### Enable adaptive behaviour
+Passing in the `adaptiveLegendBreakpoint` and `adaptiveSpacingBreakpoint` props will enable the adaptive behaviour. 
+
+<Preview>
+  <Story name="Enable adaptive behaviour">
+    <RadioButtonGroup
+      name="mybuttongroup"
+      onChange={ () => console.log('change') }
+      legend="Radio group legend"
+      ml="20%"
+      legendInline
```

Comment:
```suggestion
```
see previous comments

Reply:
Can't do this because we need to showcase the inline label changing to top aligned so it has to be set to inline